helm/mds/values.yaml

```diff
@@ -38,17 +38,35 @@ apis:
     enabled: true
     pathPrefix: /native
     version: latest
+    migration: false
   mds-policy-author:
     enabled: true
     pathPrefix: /policy-author
     version: latest
     migration: false
-  mds-config:
+processors:
+  mds-event-processor:
+    enabled: true
+    pathPrefix: /event-processor
```

Comment:
Do these need a path prefix?

Reply:
i recall you asking this awhile back. q: does the processor container(s) not require the env var PATH_PREFIX?